sky/skylet/providers/lambda_cloud/node_provider.py

```diff
@@ -172,6 +172,10 @@ def _get_internal_ip(node: Dict[str, Any]):
         self.metadata.refresh([node['id'] for node in vms])
         self._guess_and_add_missing_tags(vms)
         nodes = [_extract_metadata(vm) for vm in filter(_match_tags, vms)]
+        nodes = [
+            node for node in nodes
+            if node['status'] not in ['terminating', 'terminated']
```

Comment:
curious - is `terminating` a new state added by lambda?

Reply:
Not sure if it is newly added, but I saw the `terminating` state in the response from the lambda API during the state query. It should be added now. : )